### ISSUE TITLE
Getting error when running main.py

### ISSUE THREAD

**Issue by mapsam** [bsmithgall/prisoners-dilemma#1]
Oy! I'm getting an error right off the bat. I'm not entirely sure how to debug python so I figured I'd ask. Does this require to set up a venv at all? Here's my error:

```
Sams-MacBook-Pro:prisoners-dilemma mapsam$ python main.py
  File "main.py", line 90
    except AttributeError, NotImplementedError as detail:
                                                ^
SyntaxError: invalid syntax
```

**Comment by bsmithgall** [comment]
Can you check what version of python you are running?

**Comment by bsmithgall** [comment]
If you are on 2.6 (probably system installed python if you are on an older mac), this will be fixed by 2d32300877c0b272ee8f0480957c72aa14333f57
